https.go

```diff
@@ -533,10 +545,18 @@ func (proxy *ProxyHttpServer) NewConnectDialToProxy(httpsProxy string) func(netw
 }
 
 func (proxy *ProxyHttpServer) NewConnectDialToProxyWithHandler(
-	httpsProxy string,
-	connectReqHandler func(req *http.Request),
+  https_proxy string,
+  connectReqHandler func(req *http.Request) error,
+) func(network, addr string) (net.Conn, error) {
+	return proxy.NewConnectDialToProxyWithMoreHandlers(https_proxy, nil, nil)
```

Comment:
Here you are passing nil as the connectReqHandler, can you instead pass the input callback?

Reply:
Sorry, this happened when I fixed the merge conflicts, fixed now, thanks for pointing this out!